content/en/docs/rules/conditions.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Condition Syntax
+weight: 3
+---
+
+A condition is a boolean expression related to a single event that has been detected by Falco. You can use fields related to every supported event, but we'll focus on syscalls as they're currently the most common. The language supports boolean operators and parentheses as you'd expect. For example a condition like:
+
+```
+evt.type = execve and evt.dir = < and (proc.name = cat or proc.name = grep)
+```
+
+Will trigger for each execution of `cat` or `grep`. We'll now take a closer look at what is the meaning of those fields such as `evt.dir`, how to discover the types of events there are available and which fields go with which event type.
+
+# Syscall event types, direction and args
+
+Every syscall event will present the [`evt` field class](/docs/rules/supported-fields/#field-class-evt). Each condition that we write for those events will normally starts with a `evt.type` expression or macro; this makes a lot of sense since security rules normally consider one syscall type at a time. For instance, we may want to consider `open` or `openat` to catch suspicious activity when opening files, `execve` to inspect spawned processes and so forth. We don't have to guess the syscall name, as we can look at the complete [list of supported system calls events](/docs/rules/supported-events) or to understand which ones we can use.
+
+Every syscall has an entry event and an exit event which is shown in the `evt.dir` field, also known as the "direction" of the system call. A value of `>` indicates entry, which fires when the syscall is invoked, while `<` marks exit, meaning that the call has returned. In fact, by looking at the supported system call list we can see that our events have both entries. For example:
```

Comment:
Every syscall has an entry event and an exit event which is shown in the `evt.dir` field, also known as the "direction" of the system call. A value of `>` indicates entry, which fires when the syscall is invoked, while `<` marks exit, meaning that the call has returned. In fact, by looking at the supported system call list you can see that our events have both entries. For example: